MHacks/LeaderboardCell.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  LeaderboardCell.swift
+//  MHacks
+//
+//  Created by Connor Svrcek on 8/22/19.
+//  Copyright © 2019 MHacks. All rights reserved.
+//
+
+import UIKit
+
+class LeaderboardCell: UITableViewCell {
+    
+    // MARK: member vars
+    
+    static let identifier = "leaderboard"
+    
+    // MARK: subviews
+    
+    let positionLabel: UILabel = {
+        let pos = UILabel()
+        guard let andaleFont = UIFont(name: "AndaleMono", size: UIFont.labelFontSize) else {
+            fatalError("No AndaleMono available")
+        }
+        if #available(iOS 11.0, *) {
+            pos.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: andaleFont)
+        } else {
+            pos.font = UIFont(name: "AndaleMono", size: 20)
+        }
+        if #available(iOS 10.0, *) {
+            pos.adjustsFontForContentSizeCategory = true
+        } else {}
```

Comment:
It looks like this font logic is replicated for `positionLabel`, `nameLabel`, and `scoreLabel`, can it by functionalized?